GoodreadsEDA/all_bks.py

```diff
@@ -21,52 +21,35 @@ def get_mv_release(bk_m):
                                        WHERE book_title="{bk_m}"
                                        ;""", conn)
         return df_rel["release_date"][0]
-    except IndexError as e:
-        print(f"{e} \n")
+    except IndexError:
+        traceback.print_exc()
```

Comment:
I tried to change all of these I could find - `print(e)` doesn't print the whole stacktrace, so it's usefulness is limited. This prints the whole stacktrace.

Reply:
Awesome, certainly more informative than what I was up to. I'll make sure to do this from here on.